README.md

```diff
@@ -1 +1,25 @@
-# json-schema-form
\ No newline at end of file
+<p align="center">
+<img src=".github/media/jsf_logo_dark.png"  width="600" alt="json-schema-form">
+</p>
+
+<p align="center">
+  <code>json-schema-form</code> is a headless UI form library powered by <a href="https://json-schema.org/">JSON Schemas</a>. 
+<br/>
+  It transforms JSON schemas into Javascript to be consumed by your UI libraries.
+</p>
+
+---
+
+### Why JSON Schemas for forms?
```

Comment:
I can't find the link, is the JSON schema article published? Might be fun to include it?

Reply:
I swear I answer this comment yesterday, but it's gone. The article was not published yet. But once it is, I'll add it here!